src/app_charts/base/cloud/cert-manager-issuers.yaml

```diff
@@ -15,8 +15,11 @@ spec:
     secretName: cluster-authority
 {{ if eq .Values.certificate_provider "lets-encrypt" }}
 ---
+# While an Issuer may satisfy our current needs within the default namespace,
+# anticipating future growth and potential deployments in additional namespaces,
+# adopting a ClusterIssuer offers a more scalable and versatile solution.
```

Comment:
Why does it offer a more scalable solution? Instead of talking about future needs, maybe try specifying what a `ClusterIssuer` allows us to do that a regular `Issuer` doesn't.

Reply:
I suggested a different wording above, but marcos did not wanted to use it and I am so busy today that I merged to unblock you. Feel free to send a new PR with a better working.

Reply:
Thanks Stefan. This will allow us to test and eventually get another PR in if necessary.

Reply:
> Why does it offer a more scalable solution?

It offers a more scalable solution in the sense that more applications can be deployed using the same `ClusterIssuer`. Let's add more clarifications in a subsequent PR, I was trying to keeping it as abstract as possible.